lib/pipeline.js

```diff
@@ -104,7 +74,13 @@ class Pipeline extends Readable {
   }
 
   parseOperation (operation) {
-    return parseCode(operation, this.context, this.variables, this.basePath)
+    let result = this.loaderRegistry.load(operation, this.context, this.variables, this.basePath)
+
+    if (!result) {
+      throw new Error(`Failed to load operation ${operation.value}`)
```

Comment:
I think that the pipeline should throw if an operation fails to load rather than continuing in an invalid state